quodlibet/quodlibet/util/environment.py

```diff
@@ -32,6 +32,12 @@ def _dbus_name_owned(name):
         return False
 
 
+def is_gnome():
+    """If we are running under Gnome"""
+
+    return not is_plasma() and not is_unity() and not is_enlightenment()
```

Comment:
This is just so wrong , you can use `return os.environ.get("XDG_SESSION_DESKTOP", "") == "gnome"` instead